### DISCUSSION TITLE
LightDM doesn't start on boot

### DISCUSSION THREAD

**Discussion by Mogaba** [ltsp/ltsp#809] [Q&A] [answered]
Hi, it's me again with Alt Linux :)

I'm using this guide to make a "thin" Alt Linux client that runs freerdp:
https://github.com/ltsp/ltsp/wiki/xfreerdp

I solved almost all problems, the biggest one is that LightDM doesn't start on boot. The service is enabled and I can start it manually with `systemctl start lightdm` and it runs perfectly fine after that. 

I don't know how it should work: is LightDM started by LTSP or it's systemd's job?

Here's my ltsp.conf:
```

# /bin/sh -n
# LTSP configuration file
# Documentation=man:ltsp.conf(5)

[server]

[common]

[clients]
INCLUDE=rdp_login_screen
PAM_AUTH_TYPE=0
LOCAL_SWAP=0
DNS_SERVER="10.1.0.8 10.1.0.19"
DEBUG_LOG=1

[rdp_login_screen]
# Create a user that is local to the client, so that authentication/SSHFS/NFS aren't needed:
PRE_INIT_RDP_USER='mkdir -p /run/home && useradd --base-dir=/run/home --comment="RDP user" --create-home --shell=/bin/bash --system rdp'
# Enable autologin for that user:
AUTOLOGIN=rdp
# If autologin doesn't work with UBUNTU >= 20.04 ( https://github.com/ltsp/ltsp/discussions/281 ) try add this
RELOGIN=0
# Create an appropriate session in /etc/ltsp, then symlink it:
POST_INIT_XFREERDP_SESSION="mkdir /usr/share/xsessions; ln -s /etc/ltsp/xfreerdp.desktop /usr/share/xsessions/xfreerdp.desktop"
# And set it as the default session of this user:
LIGHTDM_CONF='user-session=xfreerdp
              autologin-user-timeout=0
              session-cleanup-script=systemctl restart lightdm'

[1e:ca:bd:4d:ad:ec]
HOSTNAME=ltsp-alt01
```

**Comment by Mogaba** [comment, marked as answer]
I got it working: because I installed X and LightDM manually in an X-less system, the default systemd target was "multi-user". I changed it to "graphical" and it did the job.

**Comment by alkisg** [comment]
So LTSP currently works in Alt Linux? Yey, a breakthrough, cheers! :)
